tests/conftest.py

```diff
@@ -43,6 +43,7 @@
     ("sqlite", "c8s"),  # conditional patching + autoconf
 ]
 
+# these packages only have a single commit and cannot be tested for updates
```

Comment:
- The info about single commit is in the variable name.
- They actually can be used for tests -- you can re-generate source-git from the same commit and that's exactly what we are doing in `test_update_from_same_commit`.

Reply:
yes, but without any context what the "single commit" means, that's why I added the comment

okay, I will update